docs/meetings/2022-09-amsterdam.md

```diff
@@ -0,0 +1,3 @@
+# EESSI Community Meeting (Sept'22, Amsterdam)
+
+*(see [https://hackmd.io/WJz9RJA6RpOcgMWc4Oym1g](https://hackmd.io/WJz9RJA6RpOcgMWc4Oym1g))*
```

Comment:
The intention is to move the contents of the HackMD note to this page, but for now we just need to have the URL in place...